docs/user-guide/how_to_use_hypernode_auto_discovery.md

```diff
@@ -0,0 +1,109 @@
+# Usage Document
+
+## Introduction
+
+This design document describes the design and implementation of the HyperNode network topology discovery feature in Volcano. This feature automatically discovers the network topology structure within the cluster and creates and maintains HyperNode custom resources (CRs) based on the discovered topology information. Consequently, the Volcano Scheduler will leverage these HyperNode CRs for scheduling decisions, eliminating the need for users to manually maintain HyperNode information.
+
+## Prerequisites
+
+Please follow [this guide](https://volcano.sh/en/docs/v1-11-0/network_topology_aware_scheduling/#installing-volcano) to install Volcano with Network Topology Aware Scheduling feature enabled.
+
+## Configuration
+
+The HyperNode network topology discovery feature is configured via a ConfigMap. The ConfigMap contains the configuration for the discovery sources, such as UFM, RoCE, and label, you can modify the configuration according to your own cluster environments.
+Please note that you should replace with your Volcano namespace if Volcano is not installed in the default namespace.
+
+### Secret Configuration (Required First Step)
+
+Before configuring the UFM discovery, you must first create a Kubernetes Secret to store your UFM credentials:
+
+```bash
+kubectl create secret generic ufm-credentials \
+  --from-literal=username='your-ufm-username' \
+  --from-literal=password='your-ufm-password' \
+  -n volcano-system
+```
+ > Note: Replace your-ufm-username and your-ufm-password with your actual UFM credentials, and adjust the namespace if needed.
```

Comment:
We can add a `> Warning` or `> Notice` here to remind users that Secrets are still unencrypted by default. If users need to encrypt Secrets, please refer to: https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/

Reply:
Already add and moved to user guide.